python/cugraph/cugraph/structure/graph_implementation/simpleGraph.py

```diff
@@ -88,7 +94,8 @@ def __from_edgelist(
         destination="destination",
         edge_attr=None,
         renumber=True,
-        legacy_renum_only=False,
+        legacy_renum_only=True,
```

Comment:
Setting this parameter to `True` skips the `C++ renumbering` and I don't think we want this by default.